src/get-started/localisation/index.md.njk

```diff
@@ -0,0 +1,101 @@
+---
+layout: layout-pane.njk
+title: Using localisation with GOV.UK Frontend
+section: Get started
+theme: How to guides
+order: 5
+description: Localisation allows you to translate GOV.UK Frontend components into other languages
+---
+
+Localisation allows you to translate GOV.UK Frontend components into other languages
+
+## Set up localisation using Nunjucks
+
+If you’re using the Nunjucks macros provided by GOV.UK Frontend, you can localise the UI text that appears within components by passing translation strings as named parameters.
+
+The individual components using these parameters include:
+
+* accordion
+* character count
+* error message
+* footer
+* header
+* summary list
+
+## Set up localisation with Javascript
+
+You can only pass translation strings to JavaScript with a list of key-value pairs.
+
+When using a list of key-value pairs, the keys must be in quote marks, for example, “YOUR_KEY”.
+
+You can also pass a `locale` property containing a [two-letter ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), which determines which plural forms and number formatting rules you use. If you do not include this, the `locale` defaults to the document language specified on the `html` element.
+
+### Passing translation strings through `initAll`
+
+You can pass translation strings for multiple components when initialising GOV.UK Frontend. All Frontend components will automatically use these translations, unless initialised separately.
+
+
+```
+window.GOVUKFrontend.initAll({
+    accordion: {
+   	 i18n: {
+   		 locale: "cy",
+   		 translations: {
+   			 show_all_sections: "Dangoswch bob adran",
+   			 hide_all_sections: "Cuddiwch bob adran",
+   			 show_this_section: "Dangoswch yr adran hon",
+   			 hide_this_section: "Cuddio'r adran hon"
+   		 }
+   	 }
+    },
+    character_count: {
+   	 i18n: {
+   		 locale: "en",
+   		 translations: {
+   			 characters_over_limit_one: "Thou have writen %{count} letter too many",
+   			 characters_over_limit_other: "Thou have writen %{count} letters too many",
+   			 characters_under_limit_one: "Thou have %{count} letter remainen",
+   			 characters_under_limit_other: "Thou have %{count} letters remainen"
+   		 }
+   	 }
+    }
+})
+```
+
+### Passing translation strings directly to a component
+
+You can use the same format when initialising components individually or collectively:
+
+```
+
+new window.GOVUKFrontend.Accordion(
+
+document.getElementById('#custom-accordion'), {
+
+i18n: {
+
+locale: 'cy',
+
+translations: {
+
+'show_all_sections': 'Dangoswch bob adran',
+
+'hide_all_sections': 'Cuddiwch bob adran'
+
+}
+
+}
+
+}).init()
+
+```
+
+### Handling plural forms
+
+Different languages have different rules for pluralisation.
```

Comment:
Could change to 'Languages have different rules for pluralisation.' to remove repetition of 'different'.